CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+## 0.18.0rc1 / 2016-04-07
+
+* [BUGFIX] Fix operator precedence in PromQL
+* [CHANGE/BUGFIX] Target identity considers path and parameters additionally to host and port
+* [CHANGE] Rename metric `prometheus_local_storage_invalid_preload_requests_total` to `prometheus_local_storage_non_existent_series_matches_total`
+* [FEATURE] Deduplicate targets within the same scrape job
+* [FEATURE] Add varbit chunk encoding (higher compression, more CPU usage – disabled by default)
+* [FEATURE] Add `holt_winters` query function
+* [FEATURE] Add relative complement `unless` operator to PromQL
+* [ENHANCEMENT] Handle data corruption more gracefully
+* [ENHANCEMENT] Quarantine series file on write issues
```

Comment:
Perhaps: "Quarantine series file if data corruption is encountered (instead of crashing)."